src/makecode/utils.ts

```diff
@@ -56,17 +60,38 @@ export const generateProject = (
 
 export const generateCustomFiles = (
```

Comment:
Are there tests for this generation? It feels like it should be pretty straightforward to test the generation (maybe snapshot testing if there's too much output?).

Reply:
There aren't at the moment, but yes, should be easy to add.

Reply:
I'm going to merge this to help folks move forward with projects that need this version bump, but please can you look at test coverage for project generation next?